requestfinder.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 SUSE LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from ConfigParser import ConfigParser
+from xdg.BaseDirectory import load_first_config
+from xml.etree import cElementTree as ET
+import sys
+import cmdln
+import logging
+import urllib2
+import osc.core
+import yaml
+import os
+
+import ToolBase
+
+makeurl = osc.core.makeurl
+
+logger = logging.getLogger()
+
+
+class Requestfinder(ToolBase.ToolBase):
+
+    def __init__(self):
+        ToolBase.ToolBase.__init__(self)
+        self.devel = None
+
+    def fill_package_meta(self, project):
+        self.package_metas = dict()
+        url = makeurl(self.apiurl, ['search', 'package'], "match=[@project='%s']" % project)
+        root = ET.fromstring(self.cached_GET(url))
```

Comment:
Should be able to pass `withfullhistory` ever since https://github.com/openSUSE/osc/commit/d401eb2c691027e8913147029a3ef2c126b2e5c4 like I do in `metrics.py` and elsewhere.

https://github.com/openSUSE/osc-plugin-factory/blob/c32b59bd554556520bdc4334c8cb6dd1b2957a6b/devel-project.py#L25-L34

Reply:
Not sure what the plan is as you removed the other references to references to `withfullhistory`. This comment should be on the `['search', 'request']` line below instead.

Reply:
the code that required withfullhistory wasn't used anymore